README.md

```diff
@@ -1 +1,81 @@
-# stem-diverse-tv-backend
\ No newline at end of file
+# stem-diverse-tv-backend
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/225705-STEM-diverse-tv)
+
+# stem-diverse-tv-backend
+
+STEM Diverse TV is a project which gather and provide inspiring, motivating, informative, educational and supportive videos about diversity in STEM. This is the backend of the project.
+
+**Table of Contents**
+
+- [Setup and run](#setup-and-run)
+    - [Run app](#run-app)
+- [Contributing](#contributing)
+- [Contact](#contact)
+- [License](#license)
+
+## Setup and run
+
+To setup the project locally follow the instructions:
+
+#### Fork
+
+_**Note**_: *This is only needed if you want to contribute to the project.*
+
+If you want to contribute to the project you will have to create your own copy of the project on GitHub. You can do this by clicking the Fork button that can be found on the top right corner of the [landing page](https://github.com/anitab-org/stem-diverse-tv) of the repository.
+
+#### Clone
+
+_**Note**_: *For this you need to install git on your machine. You can download the git tool from [here](https://git-scm.com/downloads).*
+
+ * If you have forked the project, run the following command -
+
+   `git clone https://github.com/YOUR_GITHUB_USER_NAME/stem-diverse-tv`
+
+   where `YOUR_GITHUB_USER_NAME` is your GitHub handle.
+
+ * If you haven't forked the project, run the following command -
+
+   `git clone https://github.com/anitab-org/stem-diverse-tv`
+
+ * Now after you cloned the repository, get into the anitab-org.github.io directory by -
+
+   `cd stem-diverse-tv`
+
+#### Remote
+
+_**Note**_: *This is only needed if you want to contribute to the project.*
+
+When a repository is cloned, it has a default remote named `origin` that points to your fork on GitHub, not the original repository it was forked from. To keep track of the original repository, you should add another remote named upstream. For this project it can be done by running the following command -
+
+`git remote add upstream https://github.com/anitab-org/stem-diverse-tv`
+
+You can check that the previous command worked by running `git remote -v`. You should see the following output:
+
+```
+$ git remote -v
+origin  https://github.com/YOUR_GITHUB_USER_NAME/stem-diverse-tv (fetch)
+origin  https://github.com/YOUR_GITHUB_USER_NAME/stem-diverse-tv (push)
+upstream        https://github.com/anitab-org/stem-diverse-tv (fetch)
+upstream        https://github.com/anitab-org/stem-diverse-tv (push)
+```
+
+#### Run app
+
+Download the latest stable version of the app using the link https://appcenter-filemanagement-distrib3ede6f06e.azureedge.net/3918d5a8-73a3-4e9d-a200-70b8d3526daf/com_stemdiversetv-0.1.3-12-mobile-debug.apk?sv=2019-02-02&sr=c&sig=yf2WsRZDloZeBKW%2BylgsUkorsMjdIXNyLIWLwRUm5Sc%3D&se=2020-07-25T19%3A06%3A16Z&sp=r
```

Comment:
Download the latest Android testing version of the app using the [link](https://appcenter-filemanagement-distrib3ede6f06e.azureedge.net/3918d5a8-73a3-4e9d-a200-70b8d3526daf/com_stemdiversetv-0.1.3-12-mobile-debug.apk?sv=2019-02-02&sr=c&sig=yf2WsRZDloZeBKW%2BylgsUkorsMjdIXNyLIWLwRUm5Sc%3D&se=2020-07-25T19%3A06%3A16Z&sp=r)

Reply:
should i add the splash screen or later?

Reply:
Later we need approval for colora